src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,6 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+const numArray = number.toString().split("").reverse().map(Number);
+  return numArray;
```

Comment:
Instead of assigning the method to a variable you could just return it in one line.
`````
return number.toString().split("").reverse().map(Number);
``````